paywall/src/pages/index.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react'
-import Paywall from '../components/Paywall'
-import BrowserOnly from '../components/helpers/BrowserOnly'
+import PropTypes from 'prop-types'
 
-export default function NextPaywall() {
-  return (
-    <div>
-      <BrowserOnly>
-        <Paywall />
-      </BrowserOnly>
-    </div>
-  )
+import Index from '../components/Index'
+
+export default function NextPaywall({ router }) {
```

Comment:
Let's make sure default export have a name which matches the file's name.
I think `index.js` is not a great idea (given the fact that `index.js` files are treated differently for imports. Maybe `PaywallHome.js`?

And then you would name the component `PaywallHome`?

Reply:
again, for consistency, `index.js` is used for the page that is served by next in `unlock-app`, so I am not going to change that filename. `PaywallHome` is a fabulous name.

Reply:
fixed the name, PTAL

Reply:
I really think it is important we keep the component names in sync with the file names so I submitted a PR that fixes this in unlock app: https://github.com/unlock-protocol/unlock/pull/2146
Please do the same thing here.

Reply:
roger. Done, PTAL